### PR TITLE
* : replace fmt.Sprint with strconv

### PR DESCRIPTION
**Pull request by hawkingrei**
Signed-off-by: Weizhen Wang <wangweizhen@pingcap.com>

### What problem does this PR solve?

Issue Number: ref #31716

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

**Comment by siddontang** [comment]
the reference issue doesn't show anything, so why do we do this?

**Comment by hawkingrei** [comment]
> the reference issue doesn't show anything, so why do we do this?

Updated

File: expression/generator/other_vec.go [hunk 1]
```diff
@@ -329,7 +330,7 @@ func (g inGener) gen() interface{} {
 		}
 		return *j
 	case types.ETString:
-		return fmt.Sprint(randNum)
+		return strconv.FormatInt(randNum,10)
```
Comment (kennytm) [inline expression/generator/other_vec.go:333]:
```suggestion
		return strconv.FormatInt(randNum, 10)
```

**Review by kennytm — APPROVED** [review]

**Review by qw4990 — APPROVED** [review]

**Pull request merged**